src/features/dashboard/components/ApiTokenForm/api-token.form.tsx

```diff
@@ -105,9 +106,10 @@ const ApiTokenForm = (props: HTMLAttributes<HTMLFormElement>) => {
       });
       createToken(name, selectedTokenScope);
       setFormIsCleared(true);
+      setToggleModal(!is_toggle);
```

Comment:
I think `setToggleModal(prev => !prev)` is a nicer way of handling it. If setting your state is dependent on your previous state, you should use this prev (or prevState) method.